src/PostgREST/DbStructure/Table.hs

```diff
@@ -22,6 +22,8 @@ data Table = Table
   , tableName        :: TableName
   , tableDescription :: Maybe Text
   , tableInsertable  :: Bool
+  , tableUpdatable   :: Bool
+  , tableDeletable   :: Bool
```

Comment:
Perhaps there should be a comment mentioning that these fields are not related to privileges but to view capabilities

Reply:
Yes, I was confused at first when looking at it. I think a comment won't hurt.